src/uu/tr/src/operation.rs

```diff
@@ -369,56 +373,29 @@ impl<A: SymbolTranslator, B: SymbolTranslator> SymbolTranslator for ChainedSymbo
 #[derive(Debug)]
 pub struct DeleteOperation {
     set: Vec<u8>,
-    complement_flag: bool,
 }
 
 impl DeleteOperation {
-    pub fn new(set: Vec<u8>, complement_flag: bool) -> Self {
-        Self {
-            set,
-            complement_flag,
-        }
+    pub fn new(set: Vec<u8>) -> Self {
+        Self { set }
     }
 }
 
 impl SymbolTranslator for DeleteOperation {
     fn translate(&mut self, current: u8) -> Option<u8> {
-        let found = self.set.iter().any(|sequence| *sequence == current);
-        if self.complement_flag == found {
-            Some(current)
-        } else {
-            None
-        }
-    }
-}
-
-pub struct TranslateOperationComplement {
-    iter: u8,
-    set2_iter: usize,
-    set1: Vec<u8>,
-    set2: Vec<u8>,
-    translation_map: HashMap<u8, u8>,
-}
-
-impl TranslateOperationComplement {
-    fn new(set1: Vec<u8>, set2: Vec<u8>) -> Self {
-        Self {
-            iter: 0,
-            set2_iter: 0,
-            set1,
-            set2,
-            translation_map: HashMap::new(),
-        }
+        use std::ops::Not;
```

Comment:
Please move this `use` to the top of the file.

Btw, I'm really happy how much simpler this part becomes!

Reply:
Me too, the net -89 makes me very happy